src/core/coap/coap_message.cpp

```diff
@@ -187,6 +187,23 @@ otError Message::AppendUriPathOptions(const char *aUriPath)
     return error;
 }
 
+otError Message::AppendBlockOption(Message::BlockType aType, uint32_t aNum, bool aMore, otCoapBlockSize aSize)
+{
+    otError  error   = OT_ERROR_NONE;
+    uint32_t encoded = aSize;
+
+    VerifyOrExit(aNum < 0xFFFFF, error = OT_ERROR_INVALID_ARGS);
```

Comment:
nit: magic number

Reply:
This file actually contains many definitions of similar masks. Should these eventually be changed too, or are masks generally acceptable?

https://github.com/openthread/openthread/blob/3954b549dba3fe2ad0a01b8fd8df4f41124825b7/src/core/coap/coap_message.cpp#L105
https://github.com/openthread/openthread/blob/3954b549dba3fe2ad0a01b8fd8df4f41124825b7/src/core/coap/coap_message.cpp#L169